docs/installation.md

```diff
@@ -0,0 +1,102 @@
+# Installation
```

Comment:
How about naming it `Schema Library` and put it on the Integration section next to the `UI Library` guide?

Reply:
Thank you for your advice. 👍 

Adding documentation to the section next to `UI Libraries` might be a good idea.
However, I think installation is the first thing users want to know about how to use it. So I added it to the section next to `Overview`.

I'll try adding it here first and then edit it if it's inappropriate.